beacon_node/eth2_libp2p/src/peer_manager/mod.rs

```diff
@@ -13,7 +12,8 @@ use futures::Stream;
 use hashset_delay::HashSetDelay;
 use libp2p::core::ConnectedPoint;
 use libp2p::identify::IdentifyInfo;
-use slog::{debug, error, warn};
+use peerdb::{BanOperation, BanResult, ScoreUpdateResult};
+use slog::{crit, debug, error, warn};
```

Comment:
```suggestion
use slog::{debug, error, warn};
```
this should fix the failing test